scorecard/tests/test_geography.py

```diff
@@ -94,9 +95,9 @@ def setUp(self):
 
     def test_geography_without_bbox(self):
         js_parent = serializers.GeographySerializer(
-            self.parent_geography, context={"request": None}).data
+        self.parent_geography, context={"request": None}).data
```

Comment:
I think your indentation is a bit off here

```suggestion
            self.parent_geography, context={"request": None}).data
```